.github/workflows/shared-build-deploy-container.yml

```diff
@@ -114,7 +117,7 @@ jobs:
           echo "environment=$DEPLOYMENT_ENVIRONMENT" >> $GITHUB_OUTPUT
 
   validate-event:
-    name: "Block manual releases to staging and production"
+    name: "Block workflow_dispatch for staging and production"
```

Comment:
`workflow_dispatch` in plain English means to run a workflow manually:
<https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow#configuring-a-workflow-to-run-manually>

A person reading this label must know that.
If they don't, having `workflow_dispatch` in the label is cryptic for no relevant reason.
I strongly prefer (and vote) to keep the "plain English" name here.